### PR TITLE
Add FLCO-FLYCOLORF7MINI.config

### PR DESCRIPTION
**Pull request by tobeornottobe81**
Add  FLCO-FLYCOLORF7MINI target

### CODE REVIEW

**Comment by tobeornottobe81** [comment]
image

Choose one or according to the actual product, thank you。

File: configs/default/FLCO-FLYCOLORF7MINI.config [hunk 1]
```diff
@@ -0,0 +1,149 @@
+# Betaflight / STM32F7X2 (S7X2) 4.2.0 Jun 14 2020 / 03:05:26 (8f2d21460) MSP API: 1.43
+
+#define USE_GYRO_SPI_MPU6500
+#define USE_ACC_SPI_MPU6500
+#define USE_FLASH_M25Q128JV
```
Comment (haslinghuis) [inline configs/default/FLCO-FLYCOLORF7MINI.config:5]:
```suggestion
#define USE_FLASH_W25Q128FV
```

File: configs/default/FLCO-FLYCOLORF7MINI.config [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+# master
+set mag_bustype = I2C
+set mag_i2c_device = 2
+set baro_bustype = I2C
+set baro_i2c_device = 1
+set adc_device = 3
+set blackbox_device = SPIFLASH
+set blackbox_sample_rate = 1/1
+set debug_mode = GYRO_SCALED
```
Comment (haslinghuis) [inline configs/default/FLCO-FLYCOLORF7MINI.config:128]: Remove user settings
```suggestion
```

File: configs/default/FLCO-FLYCOLORF7MINI.config [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+set dshot_burst = ON
+set serialrx_provider = SBUS
```
Comment (haslinghuis) [inline configs/default/FLCO-FLYCOLORF7MINI.config:130]: Determined by cloud API
```suggestion
```

File: configs/default/FLCO-FLYCOLORF7MINI.config [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+set current_meter = ADC
+set battery_meter = ADC
+set vbat_scale = 111
+set ibata_scale = 220
+set ibata_offset = -1300
+set beeper_inversion = ON
+set beeper_od = OFF
+set pid_process_denom = 4
```
Comment (haslinghuis) [inline configs/default/FLCO-FLYCOLORF7MINI.config:138]: Is this intended?
```suggestion
set pid_process_denom = 4
```

**Review by haslinghuis — CHANGES_REQUESTED** [review]

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**